test/integration/client/api-tests.js

```diff
@@ -15,6 +15,17 @@ suite.test('pool callback behavior', done => {
   })
 })
 
+suite.test('query timeout', (cb) => {
+  const pool = new pg.Pool({query_timeout: 1000})
+  pool.connect().then((client) => {
+    client.query('SELECT pg_sleep(2)', assert.calls(function (err, result) {
+      assert(err)
```

Comment:
can we assert what kind of error we get?

Reply:
i'll check the message.

Reply:
@kibertoad done